contractions/data/slang_dict.json

```diff
@@ -1,17 +1,46 @@
 {
-    "'aight": "alright",
-    "dunno": "do not know",
-    "howdy": "how do you do",
-    "ima": "I am going to",
-    "innit": "is it not",
-    "iunno": "I do not know",
-    "g'day": "good day",
-    "gonna": "going to",
-    "gotta": "got to",
-    "wanna": "want to",
-    "woulda": "would have",
-    "gimme": "give me",
-    "asap": "as soon as possible",
-    "u": "you",
-    "r ": "are "
+     "'aight": "alright",
+     "abt": "about",
+     "acct": "account",
+     "altho": "although",
+     "asap": "as soon as possible",
+     "avg": "average",
+     "b4": "before",
+     "bc": "because",
+     "btw": "by the way",
+     "chk": "check",
```

Comment:
have never seen this one before... you sure it is often used?

Reply:
You are referring to "chk" right? I've seen it before but not that often. I agree, it can be removed from the list.